scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -676,101 +671,61 @@ def test_stats(error, mean_max, rms_max):
     angles[:, 1] = rnd.uniform(low=0, high=np.pi, size=(n,))
     angles[:, 2] = rnd.uniform(low=-np.pi, high=np.pi, size=(n,))
 
-    for seq_tuple in permutations('xyz'):
-        # Extrinsic rotations
-        seq = ''.join([seq_tuple[0], seq_tuple[1], seq_tuple[0]])
-        rotation = Rotation.from_euler(seq, angles)
-        angles_quat = rotation.as_euler(seq)
-        angles_mat = rotation._as_euler_from_matrix(seq)
-        assert_allclose(angles, angles_quat, atol=0, rtol=1e-13)
-        assert_allclose(angles, angles_mat, atol=0, rtol=1e-9)
-        test_stats(angles_quat - angles, 1e-16, 1e-14)
-        test_stats(angles_mat - angles, 1e-15, 1e-13)
-
-        # Intrinsic rotations
+    seq = "".join([seq_tuple[0], seq_tuple[1], seq_tuple[0]])
+    if intrinsic:
         seq = seq.upper()
-        rotation = Rotation.from_euler(seq, angles)
-        angles_quat = rotation.as_euler(seq)
-        angles_mat = rotation._as_euler_from_matrix(seq)
-        assert_allclose(angles, angles_quat, atol=0, rtol=1e-13)
-        assert_allclose(angles, angles_mat, atol=0, rtol=1e-9)
-        test_stats(angles_quat - angles, 1e-16, 1e-14)
-        test_stats(angles_mat - angles, 1e-15, 1e-13)
-
-
-def test_as_euler_degenerate_asymmetric_axes():
+    rotation = Rotation.from_euler(seq, angles)
+    angles_quat = rotation.as_euler(seq)
+    angles_mat = rotation._as_euler_from_matrix(seq)
+    assert_allclose(angles, angles_quat, atol=0, rtol=1e-13)
+    assert_allclose(angles, angles_mat, atol=0, rtol=1e-9)
+    test_stats(angles_quat - angles, 1e-16, 1e-14)
+    test_stats(angles_mat - angles, 1e-15, 1e-13)
+
+
+@pytest.mark.parametrize("seq_tuple", permutations("xyz"))
+@pytest.mark.parametrize("intrinsic", (False, True))
+def test_as_euler_degenerate_asymmetric_axes(seq_tuple, intrinsic):
     # Since we cannot check for angle equality, we check for rotation matrix
     # equality
-    angles = np.array([
-        [45, 90, 35],
-        [35, -90, 20],
-        [35, 90, 25],
-        [25, -90, 15]])
-
-    with pytest.warns(UserWarning, match="Gimbal lock"):
-        for seq_tuple in permutations('xyz'):
-            # Extrinsic rotations
-            seq = ''.join(seq_tuple)
-            rotation = Rotation.from_euler(seq, angles, degrees=True)
-            mat_expected = rotation.as_matrix()
+    angles = np.array([[45, 90, 35], [35, -90, 20], [35, 90, 25], [25, -90, 15]])
```

Comment:
```suggestion
    angles = np.array([
        [45, 90, 35],
        [35, -90, 20],
        [35, 90, 25],
        [25, -90, 15]])
```

Autoformatter got a bit greedy.